packages/plot/src/marks/util/grid.js

```diff
@@ -1,13 +1,12 @@
 import { InternSet, ascending } from 'd3';
 import {
-  convertArrowArrayType,
   convertArrowColumn,
   isArrowTable
 } from '@uwdata/mosaic-core';
 
 function arrayType(values, name = 'density') {
   return isArrowTable(values)
-    ? convertArrowArrayType(values.getChild(name).type)
+    ? values.getChild(name).type.ArrayType
```

Comment:
This is not fully equivalent since we previously used Float64Array in many cases and now use Arrow's logic for array types.